api/api-server-site-urls-config-v1/src/main/java/com/thoughtworks/go/apiv1/serversiteurlsconfig/ServerSiteUrlsConfigControllerV1.java

```diff
@@ -77,19 +82,29 @@ public void setupRoutes() {
 
     String createOrUpdate(Request request, Response response) throws IOException {
         SiteUrls siteUrls = buildEntityFromRequestBody(request);
-        serverConfigService.createOrUpdateServerSiteUrls(siteUrls);
+
+        try {
+            serverConfigService.createOrUpdateServerSiteUrls(siteUrls);
+        } catch (GoConfigInvalidException e) {
```

Comment:
any particular reason for using `try-catch` instead of `exception` in the `setupRoutes`?

Reply:
not specific, will update it